tests/e2e/observers/test_balances.py

```diff
@@ -43,7 +46,12 @@ def on_completed():
 class TestBalanceManager(TestWithDBConn):
     @classmethod
     def setUpClass(cls):
+        TestWithDBConn().setUpClass()
         super().setUpClass()
+        cls.clear_db()
+
+    @classmethod
+    def clear_db(cls):
         cls.truncate_tables("native_balances", cascade=True)
         cls.truncate_tables("accounts", cascade=True)
```

Comment:
`TestWithDBConn` [implements a `clean_db` class method](https://github.com/fetchai/ledger-subquery/blob/main/tests/helpers/clients.py#L65) already. I don't think there was any good reason for me not using it in the first place.